app.go

```diff
@@ -64,6 +64,8 @@ func main() {
 	mux.Handle("/bench", handle(benchHandler, verbose))
 	mux.Handle("/api", handle(apiHandler, verbose))
 	mux.Handle("/health", handle(healthHandler, verbose))
+	mux.Handle("/environment", handle(environmentHandler, verbose))
+	mux.Handle("/env", handle(environmentHandler, verbose))
```

Comment:
```suggestion
	mux.Handle("/env", handle(environmentHandler, verbose))
```

Reply:
If you don't want to allow both I'd vote for `/environment`, because no other endpoints are abbreviated.

Reply:
> because no other endpoints are abbreviated.

We already have `/bench` instead of `/benchmark`